src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java25/InputImportModule.java

```diff
@@ -0,0 +1,8 @@
+// non-compiled with javac: reference to non-existent module 'mymodule'
```

Comment:
Please split into two inputs.
We keep non compiled as minimal as possible.

And we need javac25 job in circle CI, to make sure compiled input is actually compiled

Reply:
> Please split into two inputs.

Done.

> And we need javac25 job in circle CI, to make sure compiled input is actually compiled

Done in
- https://github.com/checkstyle/checkstyle/pull/18095